src/sass/app.scss

```diff
@@ -10,6 +10,7 @@
 @import 'base/typography';
 @import 'base/colors';
 @import 'base/cloak';
+@import 'base/fixes';
```

Comment:
If you apply this fix using a class, make sure to move the file to the `components` directory.